.github/workflows/linkcheck.json

```diff
@@ -22,6 +22,9 @@
         "pattern": [
             "127.0.0.1"
         ]
+      },
+      {
+      "pattern": "https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli"
```

Comment:
not sure if its intentional, but adding this here means that if this link 404s in the future, it won't get flagged by the link check

Reply:
I'm saying, I don't think this should be added to the ignored patterns because if this link does die in the future, it won't get flagged by the link checker. I believe we do want the link to be checked and for the CI to error if the link 404s

Reply:
This is need it to avoid 429 errors even with retries
```
Run find docs -name "*.md" | xargs -n 1 markdown-link-check -q -c .github/workflows/linkcheck.json
  find docs -name "*.md" | xargs -n 1 markdown-link-check -q -c .github/workflows/linkcheck.json
  shell: /usr/bin/bash -e {0}

  ERROR: 1 dead links found in docs/getting-started.md !
  [✖] https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli → Status: 429
Error: Process completed with exit code 123.
```